engine/src/main/java/graphql/nadel/engine/execution/Execution.java

```diff
@@ -163,45 +169,82 @@ private static NadelInstrumentationExecuteOperationParameters buildInstrumentati
     private FieldInfos createFieldsInfos() {
         Map<GraphQLFieldDefinition, FieldInfo> fieldInfoByDefinition = new LinkedHashMap<>();
 
+        Set<GraphQLOutputType> namespacedGraphqlTypes = overallSchema.getQueryType()
+                .getFieldDefinitions()
+                .stream()
+                .filter(topLevelFieldDef -> !topLevelFieldDef.getDirectives("namespaced").isEmpty())
```

Comment:
Perhaps creating an entry in `NadelDirectives` for `namespaced` is good idea. It would keep all directives that are part of the Nadel domain in a centralised, well-documented, place.